code/src/UI/Styles/ListView.xaml

```diff
@@ -261,7 +261,7 @@
         <Setter Property="VerticalContentAlignment" Value="Center"/>
         <Setter Property="KeyboardNavigation.TabNavigation" Value="Local" />
         <Setter Property="FocusVisualStyle" Value="{x:Null}" />
-        <Setter Property="AutomationProperties.Name" Value="{Binding Title}" />
```

Comment:
I think the automation.name should not be deleted, to avoid accessibility issues.

Reply:
removal was accidental - probably as part of a large merge